force-app/main/default/lwc/rd2ChangeHistory/rd2ChangeHistory.html

```diff
@@ -0,0 +1,17 @@
+<template>
+    <template if:true={isReady}>
```

Comment:
instead of using isReady here, could we just check that changeHistory is populated? i.e. `if:true={changeHistory}`?

Reply:
Ended up having to use a more complicated conditional here due to `slots` not working when wrapped with `<template if:true` . `hasMoreChanges` is when the app is ready and there are additional records - so the "View More" link should get rendered, and `hasChanges` for when there aren't more changes beyond the number of records rendered.